parser-utils/src/value_and_span.rs

```diff
@@ -280,3 +291,107 @@ pub fn value(value: edn::Value) -> Box<Parser<Input=Stream, Output=edn::ValueAnd
     // trees together, we could avoid creating garbage.
     satisfy(move |v: edn::ValueAndSpan| value == v.inner.into()).boxed()
 }
+
+pub fn keyword_map_(input: Stream) -> ParseResult<edn::ValueAndSpan, Stream>
+{
+    // One run is a keyword followed by at one or more non-keywords.
```

Comment:
I think the tests are sufficient and the comment attached to the public function are sufficient.